adflow/pyADflow.py

```diff
@@ -4516,259 +4482,244 @@ def setOption(self, name, value):
-            'outputdirectory':[str, './'],
-            'outputsurfacefamily':[str, 'allSurfaces'],
-            'writesurfacesolution':[bool,True],
-            'writevolumesolution':[bool,True],
-            'writetecplotsurfacesolution':[bool,False],
-            'nsavevolume':[int,1],
-            'nsavesurface':[int,1],
-            'solutionprecision':[str,'single'],
-            'gridprecision':[str,'double'],
-            'solutionprecisionsurface':[str,'single'],
-            'gridprecisionsurface':[str,'single'],
+            'storeRindLayer':[bool, True],
+            'outputDirectory':[str, './'],
+            'outputSurfaceFamily':[str, 'allSurfaces'],
+            'writeSurfaceSolution':[bool,True],
+            'writeVolumeSolution':[bool,True],
+            'writeTecplotSurfaceSolution':[bool,False],
+            'nSaveVolume':[int,1],
+            'nSaveSurface':[int,1],
+            'solutionPrecision':[str, ['single', 'double']],
+            'gridPrecision':[str, ['double', 'single']],
+            'solutionPrecisionSurface':[str, ['single', 'double']],
+            'gridPrecisionSurface':[str, ['single', 'double']],
             'isosurface':[dict, {}],
-            'isovariables':[list, []],
-            'viscoussurfacevelocities':[bool, True],
+            'isoVariables':[list, []],
+            'viscousSurfaceVelocities':[bool, True],
 
             # Physics Parameters
-            'discretization':[str, 'central plus scalar dissipation'],
-            'coarsediscretization':[str, 'central plus scalar dissipation'],
-            'limiter':[str, 'vanalbeda'],
-            'smoother':[str, 'runge kutta'],
-            'equationtype': [str, 'euler'],
-            'equationmode': [str, 'steady'],
-            'flowtype':[str, 'external'],
-            'turbulencemodel':[str, 'sa'],
-            'turbulenceorder':[str, 'first order'],
-            'turbresscale':[object, None],
-            'turbulenceproduction':[str, 'strain'],
-            'useqcr':[bool, False],
-            'userotationsa':[bool, False],
-            'useft2sa':[bool, True],
-            'eddyvisinfratio':[float, .009],
-            'usewallfunctions':[bool, False],
-            'useapproxwalldistance':[bool, True],
-            'eulerwalltreatment':[str, 'linear pressure extrapolation'],
-            'viscwalltreatment':[str, 'constant pressure extrapolation'],
-            'dissipationscalingexponent':[float, 0.67],
+            'discretization':[str, ['central plus scalar dissipation', 'central plus matrix dissipation', 'upwind']],
+            'coarseDiscretization':[str, ['central plus scalar dissipation', 'central plus matrix dissipation', 'upwind']],
+            'limiter':[str, ['van Albada', 'minmod', 'no limiter']],
+            'smoother':[str, ['Runge-Kutta', 'DADI']],
+            'equationType': [str, ['Euler', 'laminar NS', 'RANS']],
+            'equationMode': [str, ['steady', 'unsteady', 'time spectral']],
```

Comment:
time-spectral? Not a big deal either way

Reply:
No hyphen actually seems more common in the literature